<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventApprovalDockWidget</class>
 <widget class="QDockWidget" name="EventApprovalDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LRS-Editor Event Approval</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_event_class_name">
      <property name="text">
       <string>Event Class Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbx_event_class_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_route_name">
      <property name="text">
       <string>Route Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cbx_route_name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_event">
      <property name="text">
       <string>Event</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cbx_event"/>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_ok">
        <property name="toolTip">
         <string>Approve and save Event</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_skip">
        <property name="toolTip">
         <string>Skip to next Event without saving</string>
        </property>
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>315</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbx_route_name</tabstop>
  <tabstop>cbx_event</tabstop>
  <tabstop>pb_ok</tabstop>
  <tabstop>pb_skip</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
